api/v1/agentconfig_types_test.go

```diff
@@ -370,6 +370,28 @@ func TestAgentConfigSpecAdapter_GetRetryLimit(t *testing.T) {
 	}
 }
 
+func TestAgentConfigSpecAdapter_GetTTLSecondsAfterFinished(t *testing.T) {
+	// var testdataDefault int32 = 600
+	var testdatauserSubmitted int32 = 700
+	testCases := []struct {
+		name                    string
+		TTLSecondsAfterFinished *int32
+		expected                *int32
+	}{
+		// {name: "default", TTLSecondsAfterFinished: nil, expected: &testdataDefault},
```

Comment:
I'm going to guess that the only way to test the default that is set in the CRD is if I test it when the CRD is actually applied? :/